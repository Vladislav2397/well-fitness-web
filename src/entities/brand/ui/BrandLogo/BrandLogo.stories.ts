import { Story } from '@storybook/vue'
import { action } from '@storybook/addon-actions'

import BrandLogo, { type BrandLogoProps } from './BrandLogo.vue'
import { defineControls } from '@/shared/lib/stories'

export default {
    title: 'undefined-path/BrandLogo',
    component: BrandLogo,
    argTypes: defineControls({
        //
    }),
}

const Template: Story<BrandLogoProps> = (args, { argTypes }) => ({
    props: Object.keys(argTypes),
    components: { BrandLogo },
    methods: {
        handle: action('clicked'),
    },
    template: `<BrandLogo v-bind="$props" @click="handle" />`,
})

export const Default = Template.bind({})
Default.args = {
    //
}
