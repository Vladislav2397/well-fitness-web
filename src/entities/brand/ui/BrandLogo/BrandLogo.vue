<template lang="pug">

.b-brand-logo

</template>

<script lang="ts">
import { Component, Vue } from 'vue-property-decorator'

export type BrandLogoProps = {
    //
}

@Component
export default class BrandLogo extends Vue {
    //
}
</script>

<!-- <style lang="scss" src="./BrandLogo.critical.scss" /> -->
<!-- <style lang="scss" src="./BrandLogo.main.scss" /> -->
