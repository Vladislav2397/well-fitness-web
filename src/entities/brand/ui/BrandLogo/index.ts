import BrandLogo from './BrandLogo.vue'

export { BrandLogo }
