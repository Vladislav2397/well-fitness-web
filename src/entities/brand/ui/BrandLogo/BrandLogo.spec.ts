import { shallowMount } from '@vue/test-utils'
import BrandLogo /*, { type BrandLogoProps }*/ from './BrandLogo.vue'

describe('test BrandLogo', () => {
    test('snapshot', () => {
        const wrapper = shallowMount(BrandLogo, {
            propsData: {
                //
            } /* as BrandLogoProps*/,
        })
        expect(wrapper).toMatchSnapshot()
    })
})
