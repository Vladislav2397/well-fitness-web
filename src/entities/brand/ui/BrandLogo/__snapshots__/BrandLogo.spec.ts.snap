// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test BrandLogo snapshot 1`] = `<div class="b-brand-logo"></div>`;
